calcom/cal.com
# fix: display organizer location after booking

Fixes: https://github.com/calcom/cal.com/issues/11455
Fixes: https://github.com/calcom/cal.com/issues/12097

Added E2E Test

<img width="575" alt="Screenshot 2023-10-26 at 1 30 33 AM" src="https://github.com/calcom/cal.com/assets/53316345/45c179df-53ce-4d1a-9476-d46acf792b49">


How to test?

1) Create an event type with location In Person (Organzier Address) and add address and click save
2) Make sure display booking option is false.
3) Go to booking page
4) Make sure address is not visible on booking page.
5) Book any time slot
6) Make sure now address is visible on the success page and email
